### PR TITLE
Add Kubernetes client logging to FAQ

### PR DESCRIPTION
**Pull request by adamreese**
ref: https://github.com/helm/helm/pull/8067

### CODE REVIEW

**Review by bridgetkromhout — APPROVED** [review]

File: content/en/docs/faq/_index.md [hunk 1]
```diff
@@ -563,3 +563,13 @@ want to replace (overwrite) an existing repo, you can use `--force-update`.
 
 This is due to a breaking change for a security fix as explained in the [Helm
 3.3.2 release notes](https://github.com/helm/helm/releases/tag/v3.3.2).
+
+### Enabling Kubernetes client logging
+
+Printing log messages for debugging the Kubernetes client can be enabled using
+the [klog](https://pkg.go.dev/k8s.io/klog) flags. Using the `-v` flag to set
+verbosity level will be enough for most cases.
+
```
Comment (hickeyma) [inline content/en/docs/faq/_index.md:572]: How about adding the following for more clarity above the example?

"For example: "

**Review by hickeyma — CHANGES_REQUESTED** [review]
@adamreese Thanks for pushing the related doc PR. Looks good bar small change inline.

**Review by hickeyma — APPROVED** [review]
LGTM, thanks @adamreese

**Review by mattfarina — APPROVED** [review]
lgtm

**Pull request merged**